src/chi-vue/src/utils/utils.ts

```diff
@@ -64,3 +64,17 @@ export function copyArrayOfObjects(originArray: any[]) {
     return copyObject(object);
   });
 }
+
+export function detectChiVersion() {
+  const chiCssLink = document.querySelector("link[rel='stylesheet'][href*='/chi/']");
+
+  if (chiCssLink) {
+    const hrefAttribute = chiCssLink.getAttribute('href');
+
+    if (hrefAttribute) {
+      const version = hrefAttribute.match(/\d+\.\d+\.\d+/);
+
+      return version ? version[0] : null;
+    }
+  }
```

Comment:
There is no fallback return in the case Chi is not found or the version is not setup as expected. Please, include.

Reply:
Sounds good, addressed, thanks.